### PR TITLE
Added browserify

### PR DESCRIPTION
**Pull request by samartnik**

### CODE REVIEW

**Comment by samartnik** [comment]
Not sure about `package-lock.json` file as it was changed automatically after `npm install browserify` command.

File: package.json [hunk 1]
```diff
@@ -43,6 +44,7 @@
   "license": "MPL-2.0",
   "dependencies": {
     "bip39": "^2.5.0",
+    "browserify": "^16.1.1",
```
Comment (diracdeltas) [inline package.json]: can you add this as a `devDependency` instead? `npm install --save-dev browserify` would do it

**Comment by diracdeltas** [comment]
> Not sure about package-lock.json file as it was changed automatically after npm install browserify command.

yup, this should change whenever a new package is added or updated

File: package.json (hunk 1 above)
Comment (samartnik) [reply package.json]: Done. Again lots of changes in `package-lock.json` file.

**Comment by diracdeltas** [comment]
> added 128 packages from 188 contributors, removed 13 packages and updated 25 packages in 8.053s

wow browserify has a lot of dependencies!

waiting for travis to pass, otherwise lgtm

**Comment by diracdeltas** [comment]
@samartnik looks like there is a problem with one of the browserify dependencies which causes the `flow` tool to fail. could you add `.*/node_modules/module-deps/.*` to `.flowconfig` to fix it?

**Comment by samartnik** [comment]
Updated `.flowconfig`

**Pull request merged**